### PR TITLE
Added links to easy-to-follow polymer tutorials

### PR DESCRIPTION
**Pull request by gatanaso**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,10 @@
 ## Directories
 * [Webcomponents.org](https://webcomponents.org/)
 
+## Tutorials
+* [Build your first polymer element](https://www.polymer-project.org/2.0/start/first-element/intro)
```
Comment (limonte) [inline README.md]: `Build your first polymer element` -> `Build your first Polymer 2 element`

File: README.md [hunk 2]
```diff
@@ -21,6 +21,10 @@
...
+* [Build an app with App Toolbox](https://www.polymer-project.org/2.0/start/toolbox/set-up)
```
Comment (limonte) [inline README.md]: The official Polymer blog post [Should I Use Polymer Starter Kit or Polymer CLI](https://www.polymer-project.org/1.0/blog/2016-08-18-polymer-starter-kit-or-polymer-cli) states that **if you’re starting a new project, use Polymer CLI** (which is a part of Polymer App Toolbox). In fact, the Polymer Starter Toolkit 2 is just another template that you can use with Polymer CLI:



So, I'm in favor of not adding this misleading link.

Comment (gatanaso) [inline README.md]: Yes, that is true. The idea behind having that link is that it's an easy-to-follow tutorial :)

Comment (limonte) [inline README.md]: Makes sense, that's a useful link :+1:  I still think that 'Polymer App Toolbox' is misleading here. What do you think about changing the link text to `Build your first Polymer 2 app`?

Comment (gatanaso) [inline README.md]: Yes, sounds good!

**Comment by limonte** [comment]
Thanks for the contribution! :beers:

**Pull request merged**